Makefile

```diff
@@ -56,7 +56,9 @@ prefix_install: installables
 	mkdir -p "$(PREFIX)/Frameworks" "$(PREFIX)/bin"
 	rsync -a --delete "$(TEMPORARY_FOLDER)$(FRAMEWORKS_FOLDER)/$(OUTPUT_FRAMEWORK)" "$(PREFIX)/Frameworks/"
 	cp -f "$(TEMPORARY_FOLDER)$(BINARIES_FOLDER)/carthage" "$(PREFIX)/bin/"
-	install_name_tool -rpath "/Library/Frameworks" "@executable_path/../Frameworks/$(OUTPUT_FRAMEWORK)/Versions/Current/Frameworks/"  "$(PREFIX)/bin/carthage"
+	install_name_tool -delete_rpath "@executable_path/../Frameworks" "$(PREFIX)/bin/carthage" # Avoid duplication of "@executable_path/../Frameworks"
+	install_name_tool -rpath "/Library/Frameworks" "@executable_path/../Frameworks" "$(PREFIX)/bin/carthage"
```

Comment:
I am sorry that my opinion is difficult to understand.
I think that it is good for eliminating `/Library/Frameworks` in `prefix_install` on this PR.

> The resulting LC_RPATH ordering is not equivalent.
> The former:
> The later:

I prefer *former*.

`@executable_path/../Frameworks` means `/usr/local/Framework` on homebrew installation.
On *later* pattern, if another applications will install frameworks into `/usr/local/Framework` that `carthage` also uses, `carthage` will use from there instead of embedded in `CarthageKit.framework/Versions/Current/Frameworks`.